frontend/lib/user/repository.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+import boto3
+
+
+class UserDynamoDBRepository:
+
+    def __init__(self, endpoint_url, region_name, table_name):
+        self.endpoint_url = endpoint_url
+        self.region_name = region_name
+        self.table_name = table_name
+
+    def get_table(self):
```

Comment:
let's make this private by prefixing it with underscore. We don't want callers to be getting the table, there is no point in encapsulating it inside a repo otherwise.